server/db.js

```diff
@@ -0,0 +1,33 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: 'password',
```

Comment:
You can put your password in a separate file that is gitignore-d, then reference it here.